front-end-poc/src/Audit/types.ts

```diff
@@ -21,16 +21,18 @@ export interface IExecutionRouteParams {
   executionType: ExecutionType;
 }
 
+export interface IServiceIdentifier {
```

Comment:
I did this in preparation for using `version` more later on... not strictly necessary here but will be useful later.